CONTRIBUTING.md

```diff
@@ -21,6 +21,10 @@ This repository contains the conceptual documentation for .NET. The .NET documen
 - [.NET Compiler Platform SDK reference](https://github.com/dotnet/roslyn-api-docs)
     Issues and tasks for this repo are tracked in [dotnet/docs/issues](https://github.com/dotnet/docs/issues).
 
+### Contributing to International content
+
+Contribution for Machine Translated (MT) content are currently not accepted and deactivated for now. In an effort to improve the quality of MT content, we've transitioned to a Neural MT engine. We're focusing our feedback efforts on human translated content, which is used to train the Neural MT engine. So over time, the contributions will improve the quality of both human and machine translation. MT topics will have a disclaimer stating that part of the topic may be machine translated, and the **Edit** button won't be displayed as its disabled. 
```

Comment:
```suggestion
Contributions for Machine Translated (MT) content are currently not accepted. In an effort to improve the quality of MT content, we've transitioned to a Neural MT engine. We're focusing our feedback efforts on human translated content, which is used to train the Neural MT engine. So over time, the contributions will improve the quality of both human and machine translation. MT topics will have a disclaimer stating that part of the topic may be machine translated, and the **Edit** button won't be displayed because editing is disabled. 
```